src/features/pilot_management/store/index.ts

```diff
@@ -104,12 +118,17 @@ export class PilotManagementStore extends VuexModule {
 
   @Mutation
   private [DELETE_GROUP](payload: string): void {
-    this.Pilots.forEach((p: Pilot) => {
-      if (p.Group === payload) p.Group = ''
+    this.Pilots.forEach((p: Pilot, i) => {
+      if (p.Group === payload) {
+        p.Group = ''
+        p.SortIndex = this.Pilots.length + i
```

Comment:
This sort index is qualitative; as long as relative order is preserved, the sort/reindex below will clean up the group deletion properly, and deposit the newly-groupless pilots in their original order at the end of the "Ungrouped" group.